db/schema.rb

```diff
@@ -104,6 +104,8 @@
     t.integer  "club_id"
     t.boolean  "registration_open",       default: true
     t.text     "signature"
+    t.boolean  "unsubscribe_option"
```

Comment:
I would change the name to something more descriptive, such as "allow_unsubscribing".